base/db/R/dbfiles.R

```diff
@@ -742,11 +742,14 @@ dbfile.move <- function(old.dir, new.dir, file.type, siteid = NULL, register = F
 
 
   ### Get BETY information ###
-  bety <- dplyr::src_postgres(dbname   = 'bety',
-                              host     = 'psql-pecan.bu.edu',
-                              user     = 'bety',
-                              password = 'bety')
-  con <- bety$con
+  bety <- DBI::dbConnect(
+    RPostgres::Postgres(),
+    dbname   = 'bety',
+    host     = 'psql-pecan.bu.edu',
+    user     = 'bety',
+    password = 'bety'
+  )
+  con <- bety
```

Comment:
Small thing, but it's pretty common in the code below -- is there any reason to open the connection to a variable named `bety` and then immediately assign it to `con`, rather than just opening the connection using `con` in the first place?

 The con <- bety$con thing was always just a hack around the fact that dplyr opened connections differently than our earlier database commands (which may have been DBI? are we just reverting to our pre-dplyr usage?)